packages/frontier-plugins/plugin-vue/tsconfig.json

```diff
@@ -6,14 +6,20 @@
     "module": "commonjs",
     "outDir": "lib",
     "rootDir": "src",
+    "baseUrl": "src",
+    "paths": {
+      "@/*": [
+        "src/*"
+      ]
+    },
     "strict": true,
     "target": "es2017",
     "skipLibCheck": true,
     "esModuleInterop": true /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */,
   },
   "include": [
     "src/**/*"
-  ],
+, "../plugin-shared/src/testCommands/vue-add-commands/component/addComponent.spec.ts"  ],
```

Comment:
why is this here?

Reply:
That was added by mistake when i was testing some of the plugin specific functions. It has now been updated.